sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStore.java

```diff
@@ -180,18 +180,4 @@
      * @throws StateStoreException if key validation fails or clearing the entry fails
      */
     void clearProperty(final String key) throws StateStoreException;
-
-    /**
-     * @return true if the offers are suppressed.
-     * @throws StateStoreException
-     */
-    boolean isSuppressed() throws StateStoreException;
```

Comment:
This may solve our particular problem with suppress/revive, but it only papers over the real problem, which is that `StateStore` is too high level.  We should make it a key/value store.  Why does it need to know about tasks/statuses/properties/etc.?